### PR TITLE
Update Roadmap after the Q3 is done

### PR DESCRIPTION
**Pull request by j-rivero**
I've removed the links to open issues and development board to simplify the view of past Qs

### CODE REVIEW

File: roadmap.md
Comment (chapulina) [inline roadmap.md:18]: How about "finished" instead of "done"? Done implies that there's nothing left to do.

**Review by chapulina — COMMENTED** [review]
Thanks, Jose, this is a good idea. I think we should also add links to more migration resources once we wrap up that work. That's still going to take a couple weeks, so we can open another PR once that happens.

**Comment by j-rivero** [comment]
>  I think we should also add links to more migration resources once we wrap up that work. That's still going to take a couple weeks, so we can open another PR once that happens.

+1 , I just sent this during the community meeting not to forget to update this information. We can hold the changes until we have a more complete approach or merge this an open a new PR when ready. I leave that up yo you Louise.

**Review by chapulina — APPROVED** [review]

**Pull request merged**
